frontend/packages/kubevirt-plugin/src/components/vms/vm-console.tsx

```diff
@@ -76,11 +87,20 @@ const VMConsoles: React.FC<VMConsolesProps> = ({
       <VMIsDown />
     );
   }
-
   const vncServiceManual = (vnc && vnc.manual) || undefined;
   const rdpServiceManual = (rdp && rdp.manual) || undefined;
 
-  const desktopViewverSelector = isWindows(vm) ? (
+  if (
+    (!vnc && type === ConsoleType.VNC) ||
```

Comment:
same here. 

IMO we should show a warning here instead and allow other consoles to be selected